packages/ui/src/components/icons/icon-wrapper.svelte

```diff
@@ -1,25 +1,49 @@
-<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 -960 960 960" width="24px" fill={fillColor}><title>icon</title><path d={draw}/></svg>
+<svg style={cssDynamicStyles} xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 -960 960 960" width="24px" fill={fillColor}><title>icon {icon}</title><path d={draw}/></svg>
 
 <script lang="ts">
 import availableIcons from './icons-draw'
+// UTILS
+import { setInlineStyles } from '../../utils'
 // TYPES
+import type { AvailableIcon } from './types.icon'
 import type { ValueOf } from '@oscd-plugins/core'
 
 //====== INITIALIZATION ======//
 
 // props
 export let fillColor = 'white'
-export let icon: keyof typeof availableIcons
+export let icon: AvailableIcon
+export let isCustomIconButton = false
 
-let draw: ValueOf<typeof availableIcons>
+// local variables
+const cssFixedStyles = {
+	'svg-padding': ''
+}
 
 //====== REACTIVITY ======//
 
 $: draw = getIcon(icon)
+$: cssFixedStyles['svg-padding'] = setPadding(isCustomIconButton)
+// dynamic inline styles
```

Comment:
comment can go?

Reply:
yep ! It appears in multiple file, as I wanted to emphasis this logic, but finally I think that the names are clear enough.